app/web/features/profile/view/NewHostRequest.tsx

```diff
@@ -138,8 +138,8 @@ export default function NewHostRequest({
         )}
       </Typography>
       {error && <Alert severity="error">{error.message}</Alert>}
-      {hostError ? (
-        <Alert severity={"error"}>{hostError}</Alert>
+      {hostError?.message ? (
+        <Alert severity={"error"}>{hostError?.message}</Alert>
```

Comment:
```suggestion
      {hostError ? (
        <Alert severity={"error"}>{hostError.message}</Alert>
```
Would it not be simpler to do this so the check on `hostError` narrows the type and eliminates the need for optional chaining?

Reply:
Oh yeah that makes sense!